include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -224,6 +224,30 @@ int bt_hfp_hf_send_cmd(struct bt_conn *conn, enum bt_hfp_hf_at_cmd cmd);
 int bt_hfp_hf_cli(struct bt_conn *conn, bool enable);
 #endif /* CONFIG_BT_HFP_HF_CLI */
 
+#if defined(CONFIG_BT_HFP_HF_VOLUME)
```

Comment:
Same here, i.e. just remove this. It may be that you want to keep them for the callback struct, but even there I'd consider if you could simplify the HFP build-time configuration by removing these individual features from there.

Reply:
Updated

Reply:
Updated